test/withTheme_test.js

```diff
@@ -0,0 +1,34 @@
+import { expect } from "chai";
+import React from "react";
+
+import { withTheme } from "../src";
+import { createComponent, createSandbox } from "./test_utils";
+
+describe.only("withTheme", () => {
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("With widgets", () => {
```

Comment:
Can you add some more tests here that test the widgets, templates, and also the overriding?

Reply:
How do I even get the tests to run with changes? They keep running without any of my changes to the same file lol... (and only runs this one file, none of the other test files) I can literally delete all the tests and it still runs it - is there some weird build step involved and it only runs whatever got built?
I think I got it to work once by running cs-format and cs-check, but can't get `npm run test` to pick up any new changes to the file.

Reply:
@epicfaace added a few tests. I tried to get a custom ObjectFieldTemplate to render, but it never worked/was never called by Form =/ dunno why.